docs/reference/executionEngines.md

```diff
@@ -36,3 +36,16 @@ A "Generic DataFrame API" Action can run with different execution engines like S
 To check which execution engine was chosen, look for logs like the following:
 
       INFO  CustomDataFrameAction - (Action~...) selected subFeedType SparkSubFeed
+
+### Execution Engines vs Execution Environments

```

Comment:
Not 100% sure if everything is correct, feel free to edit/remove it if you want to update it later

Reply:
Thanks, did some rephrasing.